frontend/src/components/TravelerGroup/TravelerGroupModal.tsx

```diff
@@ -104,19 +117,30 @@ export default function TravelerGroupModal(props: TravelerGroupModalProps) {
         </Alert>
       )}
       <h1>Update TravelerGroup</h1>
-      {travelersInGroup.map((traveler) => (
-        <div key={traveler.id}>
-          <p>{traveler.name}</p>
-          <button onClick={() => handleRemoveFromList(traveler.id)}>X</button>
-        </div>
-      ))}
       <form onSubmit={handleFormSubmit}>
         <label>GroupName:</label>
+
+        {travelersInGroup.map((traveler) => (
+          <div key={traveler.id}>
+            <p>{traveler.name}</p>
+            <input
+              id={traveler.id}
+              type={"number"}
+              onChange={handleNewAmount}
+              value={`${traveler.amount}`}
+            />
+
+            <button onClick={() => handleRemoveFromList(traveler.id)}>
+              <DeleteForeverIcon />
+            </button>
+          </div>
```

Comment:
I would extract this to an own component. Then you wouldnt rely on the id of the input field to be the id of your traveller...

Reply:
If I would extract it ... then the compenent would get the id as key right?

Reply:
Yes as key because of the .map()
And the Traveller as a prop so can can access the id via the props